notebooks/official/training/xgboost_data_parallel_training_on_cpu_using_dask.ipynb

```diff
@@ -61,7 +61,7 @@
       "source": [
```

Comment:
<p>Replace Vertex AI SDK and CustomContainerTrainingJob with "Vertex AI Training"</p><p> </p><p>For steps, drop the first 2 bullets (Configure, Create)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1489/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>